### PR TITLE
Quick Start: New Visuals v2 with fix

### PR DESCRIPTION
**Pull request by momo-ozawa**
## Context
- In https://github.com/wordpress-mobile/WordPress-iOS/pull/18856 we reverted https://github.com/wordpress-mobile/WordPress-iOS/pull/18747, as we didn't have enough time to investigate / fix the root issue for some UI issues found during the CfT for WPiOS 20.0 
- This PR adds back the changes we made in https://github.com/wordpress-mobile/WordPress-iOS/pull/18747 and also fixes the issues mentioned on the CfT
  - Incorrect chevron icon aspect ratio (ref: p5T066-3gF-p2#comment-12523)
  - Ellipsis icon being displayed for the old tours (ref: p5T066-3gF-p2#comment-12533

### With Menu as default

1. Run the app
2. Enable QS for existing site
3. ✅ Check that it appears correctly and it can be dismissed
4. Enable QS for new site
4. ✅ Check that it appears correctly and it can be dismissed 

### With Home as default

1. Run the app
2. Enable QS for existing site
3. ✅ Check that it appears correctly and it can be dismissed
4. Enable QS for new site
4. ✅ Check that it appears correctly and it can be dismissed
5. ✅ Check that the chevron icon on card appears correctly

Tab | QS for New Site | QS for Existing Site
-- | -- | --
Home |  | 
Menu |  | 

## Regression Notes
1. Potential unintended areas of impact
- Quick Start cards on dashboard / menu

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Tested manually

3. What automated tests I added (or what prevented me from doing so)
- N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/QuickStartCell.swift [hunk 1]
```diff
@@ -22,6 +22,16 @@ import UIKit
     }
 
     private enum Metrics {
-        static let margins = UIEdgeInsets(top: 16, left: 8, bottom: 8, right: 8)
+        static func margins(for quickStartType: QuickStartType) -> UIEdgeInsets {
+            switch quickStartType {
+            case .undefined:
+                return .zero
```
Comment (hassaanelgarem) [inline WordPress/Classes/ViewRelated/Blog/QuickStartCell.swift]: Undefined here means this quick start started on an older version, which basically means the type is `newSite`. I think we should it treat it here this way.

File: WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/DashboardQuickStartCardCell.swift [hunk 2]
```diff
@@ -51,6 +43,38 @@ final class DashboardQuickStartCardCell: UICollectionViewCell, Reusable, BlogDas
                           properties: ["type": DashboardCard.quickStart.rawValue],
                           blog: blog)
     }
+
+    private func configureCardFrameView(for blog: Blog) {
+        switch blog.quickStartType {
```
Comment (hassaanelgarem) [inline WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/DashboardQuickStartCardCell.swift:48]: Same comment here for handling `undefined` as if it's `newSite`

Comment (hassaanelgarem) [reply WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/DashboardQuickStartCardCell.swift:48]: We should probably do the same for `String.title(for:)`. This is probably an existing bug 🙈

**Review by hassaanelgarem — COMMENTED** [review]
All looks good to me 🚀 
Just left a few comments, all related to the same thing.

File: WordPress/Classes/ViewRelated/Blog/QuickStartCell.swift (hunk 1 above)
Comment (momo-ozawa) [reply WordPress/Classes/ViewRelated/Blog/QuickStartCell.swift]: Good catch, thanks! Fixed -- f7ac1ba

File: WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/DashboardQuickStartCardCell.swift (hunk 2 above)
Comment (momo-ozawa) [reply WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/DashboardQuickStartCardCell.swift:48]: Fixed -- f7ac1ba

**Comment by momo-ozawa** [comment]
@osullivanchris Managed to find a solution to the UI bug 😄

**Comment by osullivanchris** [comment]
> @osullivanchris Managed to find a solution to the UI bug 😄

Nice job! Downloaded the build and looks resolved from my side 👍 🙌

**Review by hassaanelgarem — APPROVED** [review]

**Pull request merged**